src/control/cmd/daos_agent/procmon.go

```diff
@@ -38,6 +39,10 @@ type procMonRequest struct {
 	pid int32
 	// Whether the message is an attach or disconnect. Uses drpc method identifiers.
 	action drpc.MgmtMethod
+	// The UUID of the pool if action is poolconnect/disconnect
+	poolUUID string
+	// The UUID of the pool handle returned from pool connect if action is poolconnect/disconnect
```

Comment:
possibly a little clearer: The UUID of the pool handle associated with this request

Reply:
Fixed